modules/nodes-descheduler-configuring-strategies.adoc

```diff
@@ -47,10 +47,19 @@ spec:
        - name: "nodes"
          value: "3"
     - name: "RemoveDuplicates" <2>
+    - name: "RemovePodsHavingTooManyRestarts" <3>
+      params:
+       - name: "PodRestartThreshold"
```

Comment:
@damemi The capitalization on these parameters (taken from the upstream README [1]) is different than we have for the parameters for the LowNodeUtilization strategy. But it also uses caps for those parameters (for example `CPUThreshold` instead of `cputhreshold`). Will either way work? And should we update these docs to use `CPUThreshold`, `MemoryThreshold,` etc.?

[1] https://github.com/openshift/cluster-kube-descheduler-operator/#sample-cr

Reply:
@bergerhoffer Yeah, we changed the custom resource to just internally check all the params in lowercase ([here](https://github.com/openshift/cluster-kube-descheduler-operator/pull/108/files#diff-ad0243c09cf8e6c22f5f54bd27636f36R181)), so now any capitalization will work. Because of this I switched our sample CR to use CamelCase for readability. This applies to LowNodeUtilization's params too.

So you could even do `PoDrEsTaRtThReShOlD` as a param and it will now be treated the same, but I don't recommend that :)

Reply:
Thanks @damemi! I've updated all of the parameters to camel case then, I agree it's easier readability. I also changed `nodes` to `NumberOfNodes`. It looks like it honors both, but I'd appreciate if you could confirm.

Reply:
> I also changed nodes to NumberOfNodes. It looks like it honors both, but I'd appreciate if you could confirm.
@bergerhoffer yes, it does support both now. NumberOfNodes is preferred because it matches the upstream param